### DISCUSSION TITLE
mouseover an image to display a tooltip

### DISCUSSION THREAD

**Discussion by NielsvanHoof** [alpinejs/alpine#1112] [1. Help]
So first time trying alpine in the TALL stack and I'm having an issue. 

I have a livewire component that gets some data from it's function, I render this in a grid, when the user hover over said image some info is displayed beloning to that image in a tooltip. Now this is also where my issue lies, if only 1 image is displayed the tooltip works fine, but more then 1 image and the tooltip stops working.

[Codepen](https://codepen.io/nielsvanhoof/pen/WNoOObx)

**Comment by SimoTod** [comment]
If you are use the same variable for multiple images, clicking on one of them will trigger the click away of the other ones so it will always be false.
Each loop item should be a component defining its own x-data.
https://codepen.io/SimoTod/pen/ExNbGXp
